src/EditorFeatures/Core.Wpf/InlineRename/Dashboard/Dashboard.xaml.cs

```diff
@@ -340,6 +342,22 @@ private void Commit()
                 var notificationService = _model.Session.Workspace.Services.GetService<INotificationService>();
                 notificationService.SendNotification(ex.Message, title: EditorFeaturesResources.Rename, severity: NotificationSeverity.Error);
             }
+            catch (Exception ex) when (FatalError.ReportAndCatch(ex))
+            {
+                // Show a nice error to the user via an info bar
+                var errorReportingService = _model.Session.Workspace.Services.GetService<IErrorReportingService>();
+                if (errorReportingService is null)
+                {
+                    return;
+                }
+
+                errorReportingService.ShowGlobalErrorInfo(
```

Comment:
Just a nit/style opinion of my own: 

```suggestion
                errorReportingService?.ShowGlobalErrorInfo(
```

Reply:
Feel free to take or leave as you see fit :)